plugins/kubernetes/app/models/kubernetes/resource_status.rb

```diff
@@ -54,6 +54,7 @@ def check
           @details = "Waiting for resources (#{@pod.phase}, #{@pod.reason})"
         elsif @pod.events_indicate_failure?
           @details = "Error event"
+          @finished = true
```

Comment:
Adding pod status and failure event in details for better visible?

Reply:
both will be shown in the failure description